testsuite/unittests/test_iaf_ps_dc_accuracy.sli

```diff
@@ -83,19 +84,26 @@ M_ERROR setverbosity
 [0 min_exponent -1] Range   /hlist Set
 
 [ % time [ms]   tolerated error [mV]
- [    5.0           1e-13 ]       
- [  500.0           1e-9  ]  % error larger because of accumulation
+  [    5.0           1e-13 ]       
+  [  500.0           1e-9  ]  % error larger because of accumulation
 ] /Tlist Set                 % at very small computation step sizes
 
+% Models to be tested by this test
+[
+  /iaf_psc_alpha_ps 
+  /iaf_psc_delta_ps
+  % other precise models should be added to this list
```

Comment:
I think you could add `/iaf_psc_alpha_presc` to the list as well, the model is still part of NEST 2. (just not part of NEST 3.0).

Reply:
I agree, that model was there before. I also added `/iaf_psc_alpha_presc` to test_iaf_ps_dc_t_accuracy.sli.